workers/constructors/SharedWorker/META.yml

```diff
@@ -0,0 +1,7 @@
+links:
+    - product: chrome
+      url: https://crbug.com/655458
```

Comment:
This is an umbrella bug for WPT failures, which I don't think is useful to link.

Reply:
Reverted